## Data Availabiliity

<!-- GigaScience requires authors to deposit the dataset(s) supporting the results reported in submitted manuscripts in a publicly-accessible data repository such as GigaDB (see GigaDB database terms of use for complete details). This section should be included when supporting data are available and must include the name of the repository and the permanent identifier or accession number and persistent hyperlinks for the datasets (if appropriate). The following format is recommended:

"The dataset(s) supporting the results of this article is(are) available in the [repository name] repository, [cite unique persistent identifier]."

Submission of data into GigaDB does NOT serve as a substitution for submission of data to community-mandated databases. E.g. genome sequence data must still be submitted to GenBank, ENA, or DDBJ. For a complete list of community available databases please see the Reporting Standards section of our Editorial Policies.

Following the Joint Declaration of Data Citation Principles, where appropriate we ask that the data sets be cited where it is first mentioned in the manuscript, and included in the reference list. If a DOI has been issued to a dataset please always cite it using the DOI rather than the less stable URL the DOI resolves to (e.g., http://dx.doi.org/10.5524/100044 rather than http://gigadb.org/dataset/100044). For more see:

Data Citation Synthesis Group: Joint Declaration of Data Citation Principles. Martone M. (ed.) San Diego CA: FORCE11; 2014 [https://www.force11.org/datacitation]

A list of available scientific research data repositories can be found in res3data and FAIRsharing.

A Data availability statement is required for all papers, and the code snapshots should be cited here. -->

Within OpenPedCan (OPC), we harmonized, aggregated, and analyzed data from multiple sources.
We harmonized data from the Therapeutically Applicable Research to Generate Effective Treatments (TARGET cohort) Initiative, an NCI-funded collection of disease-specific projects that seeks to identify the genomic changes of pediatric cancers [@dbgap:phs000218.v23.p8].
We included already harmonized neuroblastoma samples from the Gabriella Miller Kids First (GMKF cohort) Pediatric Research Program, a large-scale effort to accelerate research and gene discovery in pediatric cancers and structural birth defects [@dbgap:phs001436.v1.p1].
Additionally, we re-harmonized all samples from the Open Pediatric Brain Tumor Atlas (OpenPBTA, PBTA cohort), an open science initiative led by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab and the Center for Data-Driven Discovery (D3B) at the Children’s Hospital of Philadelphia (CHOP), which genomically characterized pediatric brain tumor data from the Children’s Brain Tumor Network (CBTN), and the Pacific Pediatric Neuro-oncology Consortium (PNOC) [@dbgap:phs002517.v1.p1; @doi:10.1016/j.xgen.2023.100340].
Building on the work of OpenPBTA, OPC contains additional data from the following studies:

- PBTA X01 [@https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick]
- Chordoma Foundation [@https://www.chordomafoundation.org]
- MI-ONCOSEQ Study [@https://www.pathology.med.umich.edu/mctp/mi-oncoseq-study], donated to CBTN by the University of Michigan, to the PBTA cohort
- Children's Hospital of Philadelphia (CHOP) P30 Panel data generated by CHOP's Division of Genomic Diagnostics (DGD cohort) which includes targeted DNA and fusion panel data [@dbgap:phs002517.v1.p1].
- National Cancer Institute's Clinical Proteomic Tumor Analysis Consortium (CPTAC) PBTA proteomics study [@doi:10.1016/j.cell.2020.10.044]
- CPTAC adult GBM proteomics study [@doi:10.1016/j.ccell.2021.01.006]
- Project HOPE adolescent and young adult high-grade glioma proteomics study (in preparation for publication)

In addition to pediatric cancer data, OpenPedCan contains adult data from large science consortia as references.
For normal gene expression, GTEx [@dbgap:phs000424.v9.p2] was used, and for comparison with adult cancers, The Cancer Genome Atlas (TCGA) [@https://www.cancer.gov/tcga] was included.

Merged summary files for the latest release of OpenPedCan are openly accessible in [CAVATICA](https://cavatica.sbgenomics.com/u/cavatica/opentarget) or via `download-data.sh` script in the [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) repository.
Cancer group summary data from release v11 are visible within the NCI's pediatric [Molecular Targets Platform](https://moleculartargets.ccdi.cancer.gov/). 
Cohort, cancer group, and individual data are visible within [PedcBioPortal](https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpedcan_v15)


